Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -371,7 +371,7 @@ def get_size_data(self):
 
     resolution = 256
 
-    CurveSymbols = np.array("o x t + d s t2 t3 p h star ?".split())
+    CurveSymbols = np.array("o x t + d star t2 t3 p h s ?".split())
```

Comment:
Do we ever need more than 6 symbols. If not, how about removing the rest (apart from '?')?
`CurveSymbols = np.array("o x t + d star".split())`

Reply:
I guess this should be safe, yes.

After this, I also changed `MAX_SHAPES = 6` to `MAX_SHAPES = len(OWScatterPlotBase.CurveSymbols) - 1`.